editor/editor_settings.cpp

```diff
@@ -781,6 +781,7 @@ void EditorSettings::_load_godot2_text_editor_theme() {
 	_initial_set("text_editor/theme/highlighting/word_highlighted_color", Color(0.8, 0.9, 0.9, 0.15));
 	_initial_set("text_editor/theme/highlighting/number_color", Color(0.92, 0.58, 0.2));
 	_initial_set("text_editor/theme/highlighting/function_color", Color(0.4, 0.64, 0.81));
+	_initial_set("text_editor/theme/highlighting/global_function_color", Color(0.6, 0.6, 0.9));
```

Comment:
As this only applies to the `GDScriptHighlighter` it should not be defined here nor in `editor_themes` only inside `GDScriptSyntaxHighlighter::_update_cache` with `EDITOR_DEF`. It also needs to be placed under the `gdscript` subsection.

Reply:
I wasn't sure it would only apply to the GDScript highlighter. I'll adjust these things then.